service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -69,10 +69,11 @@ public String getHearingTime(HearingBooking hearingBooking) {
         return hearingTime;
     }
 
-    public Optional<String> getHearingDateIfHearingsOnSameDay(HearingBooking hearingBooking) {
+    public Optional<String> getHearingDate(HearingBooking hearingBooking) {
         String hearingDate = null;
 
-        // If they aren't on the same date return nothing
+        // If they are on same day, then return formatted date
+        // and If they aren't on the same day, then return nothing
```

Comment:
Do we need this comment or is the code self explanatory? I will leave this to your discretion :)

Reply:
This was one the point in tech debt , that's why added it :)

Reply:
cool lets leave then :)